compiler/rustc_interface/src/tests.rs

```diff
@@ -605,7 +605,7 @@ fn test_codegen_options_tracking_hash() {
     tracked!(debug_assertions, Some(true));
     tracked!(debuginfo, DebugInfo::Limited);
     tracked!(embed_bitcode, false);
-    tracked!(force_frame_pointers, Some(false));
+    tracked!(force_frame_pointers, FramePointer::Always);
```

Comment:
Should this be MayOmit?

Reply:
Any value other than the default is correct here. This tests that changing the value of the option changes the dep tracking hash.

Reply:
and it seemed like it would be a cold day in hell before `FramePointer::Always` was the default.

Reply:
Oh Some(false) vs None